sklearn/tests/test_cross_validation.py

```diff
@@ -1250,3 +1254,30 @@ def test_cross_val_predict_sparse_prediction():
     preds_sparse = cval.cross_val_predict(classif, X_sparse, y_sparse, cv=10)
     preds_sparse = preds_sparse.toarray()
     assert_array_almost_equal(preds_sparse, preds)
+
+
+def test_rfe_cv():
+    iris = load_iris()
+    number_groups = 4
+    groups = np.floor(np.linspace(0, number_groups, len(iris.target)))
+    X = iris.data
+    y = (iris.target > 0).astype(int)
+
+    # test basic cv
```

Comment:
Why is this needed? Isn't it tested elsewhere?

Reply:
Maybe [here](https://github.com/scikit-learn/scikit-learn/blob/51c8c1619713169e61fe6dc081790ff0cb9016d5/sklearn/tests/test_metaestimators.py#L41) - but from looking at it, I'm not totally sure. @jnothman - do you know?